cursorless-talon/src/actions/find.py

```diff
@@ -6,6 +6,6 @@
 def run_find_action(targets: dict):
     """Find text in editor"""
     texts = get_text(targets, ensure_single_target=True)
-    actions.user.vscode("actions.find")
+    actions.user.fs_run_command("actions.find")
```

Comment:
This won't work outside of vscode, because the name of the command is vscode-specific.  Other editors also might not even need hacks like this one. cc/ @AndreasArvidsson

Reply:
We should:

1. Move this function to a new file called `cursorless_vscode`
2. Make it an action
3. Have the action only exist on a VSCode context

For future work:

4. Have a global version of this action that will actually run the action in Cursorless for editors that don't need a hack

Reply:
Think I've done this.